tests/components/konnected/test_init.py

```diff
@@ -706,7 +706,7 @@ async def test_state_updates_zone(
     resp = await client.post(
         "/api/konnected/device/112233445566",
         headers={"Authorization": "Bearer abcdefgh"},
-        json={"zone": "5", "temp": 32, "addr": 1},
+        json={"zone": "5", "temp": 32.0, "addr": 1},
```

Comment:
This test was relying on the temp conversion logic to convert int to float instead of passing a float